int32 id
float32[] emg
float32[] glove
float32[] glove_low_dim
int32 label
